server/server.go

```diff
@@ -3349,13 +3363,19 @@ func getNextMsg(cs *stores.ChannelStore, nextSeq, lastSent *uint64) *pb.MsgProto
 	}
 }
 
-func (s *StanServer) getSequenceFromStartTime(cs *stores.ChannelStore, startTime int64) uint64 {
-	return cs.Msgs.GetSequenceFromTimestamp(startTime)
+func (s *StanServer) getSequenceFromStartTime(cs *stores.ChannelStore, startTime int64) (uint64, error) {
+	tm, err := cs.Msgs.GetSequenceFromTimestamp(startTime)
+	if err != nil {
+		s.log.Errorf("Unable to get sequence from start time: %v", err)
+		return 0, err
```

Comment:
Is this potential for double logging since we are logging here *and* propagating the error where it might be logged higher in the stack?